charts/stable/local-ai/values.yaml

```diff
@@ -49,7 +49,8 @@ localai:
   debug: true
   cors: true
   cors_allow_origins: "*"
-  # galleries: [{"name":"model-gallery", "url":"github:go-skynet/model-gallery/index.yaml"}]
+  galleries: '[{"name":"model-gallery", "url":"github:go-skynet/model-gallery/index.yaml"}]'
```

Comment:
Lets make this a list of dicts
```yaml
galleries:
  - name: blabla
     url: abc
```
Same for preload_models

In the env use the `{{ toJson .Values.localai.galleries }}